### ISSUE TITLE
Highlight bus routes.

### ISSUE THREAD

**Issue by DFyson** [osmandapp/Osmand#1361]
Determining where a specific bus goes by highlighting its route would be very useful. Currenty all the bus routes are the same colour and it's hard to follow a route when they overlap.
Maybe this could be done by selecting a bus stop and selecting one of the routes from that?

**Comment by tbicr** [comment]
Anyway will be useful just show available bus/trams routes on selected stop: you can be in one place and need going to another, some info can be displayed on transport routes on high zoom, but easily just to know what going to destination place on stop tap.

**Comment by sonora** [comment]
Implemented.
